docs/earthfile/earthfile.md

```diff
@@ -442,6 +442,32 @@ COPY test/* .
 COPY --dir test .
 ```
 
+One can also copy from other Earthfile targets:
+
+```
+FROM alpine:3.13
+vault:
+    RUN echo aGVsbG8= > secret-data
+    SAVE ARTIFACT secret-data
+example:
+    COPY +vault/secret-data .
+    RUN cat secret-data | base64 -d
```

Comment:
I would maybe not call this "secret-data", just so we don't encourage the placement of secrets in layers as part of the official docs. Perhaps use `dummy-artifact` instead?

Reply:
lol.... yeah good point.